products/bigquery/inspec.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Inspec::Config
+manifest: !ruby/object:Provider::Inspec::Manifest
+  summary: 'InSpec resources for verifying GCP infrastructure'
+  description: |
+    InSpec resources for verifying GCP infrastructure
+overrides: !ruby/object:Overrides::ResourceOverrides
```

Comment:
What I'm about to suggest might be total overkill.

In Ansible, I've got datasources + regular modules. I actually keep two separate sets of overrides for them. We already have a way to exclude resources with `exclude`. If you had two sets of overrides, you could override them individually using `exclude`.

If you see plural + singular resources diverging, this could be valuable?

Reply:
Wait, `exclude_plural` is a property override. That makes that note much less relevant.